draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1181,7 +1181,7 @@ Repeat(10,
 Update(M0, M1)
```

Comment:
Could add in `erase expected_tag` after `erase msg` in Authenticated Decryption, as discussed for AEGIS-128L/-256.

Reply:
Done.